lib/git.rb

```diff
@@ -111,6 +111,13 @@ def self.current_branch()
       ref.split('/').last
    end
 
+   # Deletes the current branch. For cleaning up after errors.
+   def self.delete_current_branch()
+      branchName = Git::current_branch
+      Git::switch_branch('master')
```

Comment:
I think it'd be slightly better to switch to the development branch that's specified in the git config. You can get that wit `get_branch('development')`.

I also think that it'd be nice to make sure that the current branch isn't equal to the development branch, otherwise the git command will fail, since you can't delete the branch you're currently on.